src/index.js

```diff
@@ -38,6 +40,10 @@ function handleExtendsNodes(tree, options, messages) {
       throw getError(errors.EXTENDS_NO_SRC);
     }
 
+    let data = extendsNode.attrs.data ? JSON.parse(extendsNode.attrs.data) : {};
```

Comment:
here it would be nice to wrap in tray / catch because there may not be an object